docs/dev/feedback-json.rst

```diff
@@ -0,0 +1,57 @@
+.. _feedback-json-doc:
+
+Feedback JSON Documentation
+===========================
+
+Submission JSON
+---------------
+
+To submit recording feedback via our API (see: :doc:`api`), ``POST`` a JSON document to
+the ``recording-feedback`` endpoint.
+
+A sample feedback may look like::
+
+    {
+        "recording_msid": "d23f4719-9212-49f0-ad08-ddbfbfc50d6f",
+        "score": 1
+    }
+
+Score can have one of these three values:
+
+- ``1``: Mark the track as loved
+
+   - Indicates user has marked the recording as loved
+
+- ``-1``: Mark the track as hated
+
+   - Indicates user has marked the recording as hated
+
+- ``0``: Remove the feedback from the track
+
+   - Indicates user wants to remove the feedback (loved or hated) from the recording
+
+
+Fetching feedback JSON
+----------------------
+
+The JSON documents returned from our API for recording feedback look like the following::
+
+{
+   count: 1,
+   feedback: [
+      {
+         "user_id": "-- the MusicBrainz ID of the user --",
+         "recording_msid": "d23f4719-9212-49f0-ad08-ddbfbfc50d6f",
+         "score": 1
+      },
+      "-- more feedback data here ---"
+   ],
+   offset: 0,
+   total_count: 1
+}
+
+The number of feedback in the document are returned by the top-level ``count`` element. The total number of
+feedback for the user/recording are returned by the top-level ``total_count``. ``offset`` speicifies the
```

Comment:
typo "speicifies"

Reply:
Better wording for stupid english: "The total number of feedback _items_ for the"

Reply:
Done